# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/tests/mochitest/common.js
# Commit: 786805a2cdcc
# Full Hash: 786805a2cdcc4cdab632a955dace417062d0087e
# Author: James Teh <jteh@mozilla.com>
# Date: 2023-11-30 04:12:04
# Regressor Bug: 1862802
# File Overlap Count: 1
# Description:
#   Bug 1862802 part 5: Remove virtual cursor change event. r=eeejay
#   
#   Depends on D192645
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D192646
# ==============================================================================

diff -r effe12f2120a -r 786805a2cdcc accessible/tests/mochitest/common.js
--- a/accessible/tests/mochitest/common.js	Tue Nov 28 22:03:20 2023 +0000
+++ b/accessible/tests/mochitest/common.js	Tue Nov 28 22:03:21 2023 +0000
@@ -24,8 +24,6 @@
 const nsIAccessibleTextChangeEvent = Ci.nsIAccessibleTextChangeEvent;
 const nsIAccessibleTextSelectionChangeEvent =
   Ci.nsIAccessibleTextSelectionChangeEvent;
-const nsIAccessibleVirtualCursorChangeEvent =
-  Ci.nsIAccessibleVirtualCursorChangeEvent;
 const nsIAccessibleObjectAttributeChangedEvent =
   Ci.nsIAccessibleObjectAttributeChangedEvent;
 const nsIAccessibleAnnouncementEvent = Ci.nsIAccessibleAnnouncementEvent;